ethereum/networks/src/main/java/tech/pegasys/teku/networks/Eth2NetworkConfiguration.java

```diff
@@ -905,6 +907,21 @@ private Builder applyHoleskyNetworkDefaults() {
               "enr:-LS4QG0uV4qvcpJ-HFDJRGBmnlD3TJo7yc4jwK8iP7iKaTlfQ5kZvIDspLMJhk7j9KapuL9yyHaZmwTEZqr10k9XumyCEcmHYXR0bmV0c4gAAAAABgAAAIRldGgykGm32XQEAXAAAAEAAAAAAACCaWSCdjSCaXCErK4j-YlzZWNwMjU2azGhAgfWRBEJlb7gAhXIB5ePmjj2b8io0UpEenq1Kl9cxStJg3RjcIIjKIN1ZHCCIyg");
     }
 
+    private Builder applyEphemeryNetworkDefaults() {
+      return applyTestnetDefaults()
+              .constants(EPHEMERY.configName())
+              .startupTimeoutSeconds(120)
+              .trustedSetupFromClasspath(MAINNET_TRUSTED_SETUP_FILENAME)
+              .eth1DepositContractDeployBlock(0)
+              .defaultInitialStateFromUrl(
+                      "https://checkpoint-sync.ephemery.ethpandaops.io/eth/v2/debug/beacon/states/finalized")
```

Comment:
@taxmeifyoucan please help me confirm if this URL is correct

Reply:
That should work 👍